### PR TITLE
fix the inconsistent uri issue for set breakpoint request

### PR DESCRIPTION
**Pull request by testforstephen**
Signed-off-by: Jinbo Wang <jinbwan@microsoft.com>

Handle the vscode bug: https://github.com/Microsoft/vscode/issues/36471

### CODE REVIEW

**Comment by yaohaizh** [comment]
Will this target 0.2.0 release?

**Comment by testforstephen** [comment]
i prefer to include it in 0.2.0

File: com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/AdapterUtils.java [hunk 1]
```diff
@@ -220,4 +222,18 @@ public static String getSHA256HexDigest(String content) {
...
+
+    /**
+     * decode the uri string.
+     * @param uri
+     *          the uri string
+     * @return the decoded uri
+     */
+    public static String decodeURIComponent(String uri) {
+        try {
+            return URLDecoder.decode(uri, "UTF-8");
```
Comment (yaohaizh) [inline com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/AdapterUtils.java]: Please use the constant String definition if can.  
```
public static String ... 
```

File: com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/AdapterUtils.java [hunk 2]
```diff
@@ -220,4 +222,18 @@ public static String getSHA256HexDigest(String content) {
         }
         return buf.toString();
     }
+
+    /**
+     * decode the uri string.
```
Comment (yaohaizh) [inline com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/AdapterUtils.java]: `decode` ==> `Decode`

**Pull request merged**